libpod/copy_data.go

```diff
@@ -550,3 +574,91 @@ func (c *CopyData) MakeImageRef(manifestType string, compress archive.Compressio
 	}
 	return ref, nil
 }
+
+// GetData gets the Data for a container with the given name in the given store.
+func (r *Runtime) GetData(name string) (*Data, error) {
+	img, err := r.GetImage(name)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error reading image %q", name)
+	}
+
+	imgRef, err := r.GetImageRef("@" + img.ID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error reading image %q", img.ID)
```

Comment:
could we make this error message be slightly different than the one at 582?  Perhaps "error reading image reference %q"?

Reply:
fixed.